src/prompts/github.ts

```diff
@@ -0,0 +1,154 @@
+${stringify(releases)}
+
+Check:
+${stringify(mapCheckToContextValue(check))}
+
+Check Script:
+${check.script}
+
+Check Result:
+${checkResult}`,
+    promptConfig(),
+  ];
+}
+
+export function generateFindRelevantDeploymentsPrompt(
+  check: Check,
+  checkResult: string,
+  deployments: GithubDeploymentForPrompt[],
+): [string, PromptConfig] {
+  return [
+    `Based on the following deployments, which ones are most relevant to the check state change? Analyze the check script, result and releases to determine which releases are most relevant. Provide a list of deployment ids that are most relevant to the check.
+
+Deployments:
+${stringify(deployments)}
+
+Check:
+${stringify(mapCheckToContextValue(check))}
+
+Check Script:
+${check.script}
+
+Check Result:
+${checkResult}`,
+    promptConfig(),
+  ];
+}
+
+export function generateReleaseHeadlinePrompt(
+  prevRelease: string,
+  currentRelease: string,
+  diff: string,
+): [string, PromptConfig] {
+  return [
+    `The following diff describes the changes between ${prevRelease} and ${currentRelease}.
+
+Summarize the changes in a single sentence:
+${JSON.stringify(diff)}
+
+Do not describe the outer context as the developer is already aware.
+Do not yap.
+Do not use any formatting rules.`,
+    promptConfig(),
+  ];
+}
+
+export function generateReleaseSummaryPrompt(
+  prevRelease: string,
+  currentRelease: string,
+  diff: string,
+): [string, PromptConfig] {
+  return [
+    `The following diff describes the changes between ${prevRelease} and ${currentRelease}.
+
+  Summarize the changes so that another developer quickly understands what has changes:
+${diff.slice(0, MAX_DIFF_LENGTH)}.
+
+Do not describe the outer context as the developer is already aware.
+Do not yap.
+Format titles using *Title*, code using \`code\`.
+Do not use any other formatting rules.
+Focus on potential impact of the change and the reason for the change.`,
+    promptConfig(),
+  ];
+}
+
+export function generateDeploymentSummaryPrompt(
+  prevSha: string,
+  currentSha: string,
+  diff: string,
+): [string, PromptConfig] {
+  return [
+    `The following diff describes the changes between ${prevSha} and ${currentSha}.
+
+  Summarize the changes so that another developer quickly understands what has changes:
+  ${diff}
+
+  Do not describe the outer context as the developer is already aware.
+  Do not yap. Format titles using *Title*, code using \`code\`. Do not use any other formatting rules.
+  Focus on potential impact of the change and the reason for the change.`,
+    promptConfig(),
+  ];
+}
+
+export function generateFindRepoPrompt(
+  userPrompt: string,
+  allRepos: GithubRepoForPrompt[],
+): [string, PromptConfig] {
+  return [
+    `Based on the following prompt: ${userPrompt} and the list of repositories
+
+${JSON.stringify(allRepos)}
+
+Select the repository that is most relevant to the prompt.`,
+    promptConfig(),
+  ];
+}
+
+export function generateTimeframePrompt(): [string, PromptConfig] {
```

Comment:
It looks like it's not yet used in the tool

Reply:
@jan-osch used by `src/github/agent.ts`